### PR TITLE
New Topic: Social Engineering

### PR DESCRIPTION
**Pull request by pran-p**
In this article I give a brief introduction to social engineering

### CODE REVIEW

**Comment by anumehaagrawal** [comment]
@salman-bhai  Should I merge this ?

**Comment by anumehaagrawal** [comment]
@pran-p  Please fix merge conflicts

File: blog_src/_posts/2019-01-17-social-engineering.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+description: 'A brief introduction to Social Engineering'
+tags:
+- Social engineering
+- Security
+categories:
+- Compsoc
+github_username: 'pran-p'
+---
+
+# Social Engineering
```
Comment (salman-bhai) [inline blog_src/_posts/2019-01-17-social-engineering.md]: This can be removed!

File: blog_src/_posts/2019-01-17-social-engineering.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Have you ever stared into the keyboard while your friend is typing a password? Or have you ever made up stories to play a prank on your friend? If the answer to any of the above questions is yes then you are a **social engineer!**
```
Comment (salman-bhai) [inline blog_src/_posts/2019-01-17-social-engineering.md]: Kindly change it to `Social Engineer!`

**Comment by salman-bhai** [comment]
@rohit-mp kindly re run the `jekyll build` on this and push with an extra commit

**Comment by rohit-mp** [comment]
@salman-bhai should I update the date to the last weekend or keep the same?

**Comment by salman-bhai** [comment]
Please resolve merge conflicts.

**Pull request closed without merging**